src/Main.cpp

```diff
@@ -13,20 +16,26 @@ void printThreadExample(void* param) {
     (void)param;
     const int toggleDelay = 2;
 
+    BSP bsp = BSP();
+    bsp.initChip();
+
     UserInterface ui = UserInterface();
     Logger logger = Logger(LogLevel::Debug, ui);
 
+    BittyBuzzBytecode bytecode = BittyBuzzFactory::createBittyBuzzBytecode(logger);
+    auto functionRegisters = BittyBuzzFactory::createBittyBuzzFunctionRegisters();
```

Comment:
I though we said we would avoid auto. Thing is that the type here has the length of the std::array, just wanted to avoid changing it at two places at the same time. I'll change it.